PartyWebApp/Models/Attendee.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PartyWebApp.Models
+{
+    public class Attendee
+    {
+        public int id { get; set; }
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+        [DataType(DataType.Date)]
+        public DateTime birthDate { get; set; }
+        public string email { get; set; }
+        public string phoneNumber { get; set; }
```

Comment:
לא עדיף int ?

Reply:
No,
In serial numbers such as phone, id, or credit card it is better to use string instead of int or double.
The reason for it is so you'll have more flexibility.
You can't start an int with '0' like every mobile number on the planet because it will drop it, you cant save 02 because it will convert it into 2.
And in with string, you have Regex capabilities, which you should Google.